regression/bedrock.hcl

```
@@ -2,47 +2,43 @@
 version = "1.0"
 
 train {
-    step train_regression {
-        image = "python:3.9"
-        install = [
-            "pip install --upgrade pip",
-            "pip install -r requirements.txt",
-        ]
-        script = [{sh = ["python train.py"]}]
-        resources {
-            cpu = "4"
-            memory = "4G"
-        }
+  step "train_regression" {
+    image = "python:3.9"
```

```suggestion
    image = "python:3.7"
```

are we using python 3.7 everywhere?

---

no, it's up to the author of the example